### PR TITLE
Add harpy recipe

### PR DESCRIPTION
**Pull request by pdimens**
I would like to add a recipe for the Harpy haplotagging pipeline for biological sequence data

https://github.com/pdimens/HARPY

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by pdimens** [comment]
Is there something wrong with my meta.yaml file? The build section has the correct compilers, but the build seems to fail because gcc isnt found.

**Comment by dpryan79** [comment]
The is no binary called gcc. Packages need to respect standard environment variables like CC or CXX.

**Comment by pdimens** [comment]
@BiocondaBot please add label

File: recipes/harpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  name: harpy
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/pdimens/harpy/archive/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
```
[the remark this answers is not in the document]
Comment (pdimens) [reply recipes/harpy/meta.yaml]: Forgive me, I'm relatively new to this. The package does not compile anything during build. The test fails saying I need a `noarch` section, but the documentation suggests I need to use either `noarch: generic` or `noarch: python`, and I'm genuinely confused as to the right path to continue forward.

[the remark this answers is not in the document]
Comment (pdimens) [reply recipes/harpy/meta.yaml]: The only other option is `noarch: python` ? Is that what I should be using?

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
This is ready to merge if you're happy with things.

**Comment by pdimens** [comment]
Yes, thank you very much for your assistance 🙏

**Pull request merged**